model.js

```diff
@@ -1,7 +1,19 @@
+const { DataRowMessage } = require("pg-protocol/dist/messages")
```

Comment:
I think this line can be removed it's probably something VSCode has added without us asking it to